ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -96,3 +100,11 @@ Run Codeflare ODH Test
     IF    ${result.rc} != 0
         FAIL    ${TEST_NAME} failed
     END
+
+Restart Kueue
```

Comment:
Probably we can move this Method to [DistributedWorkloads.resource](https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource) and can reuse it for DW UI [Ray test](https://github.com/red-hat-data-services/ods-ci/blob/f756474e2f9a449de52568d9f70e1f6f8d3cde49/ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot#L135) workaround  as well